contentStyle.json

```diff
@@ -0,0 +1,128 @@
+		"trailingEmptyLine": "always",
+		"checkCopyright": true,
+		"copyrightBlacklist": ["data/tooltips.txt"],
+		"copyrightFormats": [
+			{
+				"holder": "^# (Based on earlier text copyright|Copyright) \\(c\\) \\d{4}(?:(?:-|, )\\d{4})? (by )?.+$",
+				"notice": [
+					"#",
+					"# Endless Sky is free software: you can redistribute it and/or modify it under the",
+					"# terms of the GNU General Public License as published by the Free Software",
+					"# Foundation, either version 3 of the License, or \\(at your option\\) any later version\\.",
+					"#",
+					"# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY",
+					"# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A",
+					"# PARTICULAR PURPOSE\\. See the GNU General Public License for more details\\.",
+					"#",
+					"# You should have received a copy of the GNU General Public License along with",
+					"# this program\\. If not, see <https\\://www\\.gnu\\.org/licenses/>\\.",
+					""
+				]
+			}
+		],
+		"regexChecks": [
+			{
+				"excludedNodes": [],
+				"checks":[
+					{
+						"description": "multiple consecutive space characters",
+						"regex": "(?<!^tip .*)([^ ])  +([^# ])",
+						"correction": {
+							"replaceWith": "\\1 \\2"
+						}
+					},
+					{
+						"description": "mixed indentation inside text",
+						"regex": "([\"'`]\t+) +",
+						"correction":{
+							"replaceWith": "\\1"
+						}
+					},
+					{
+						"description": "trailing whitespace",
+						"regex": "([^\\s])\\s+$",
+						"correction": {
+							"replaceWith": "\\1"
+						}
+					},
+					{
+						"description": "non-ASCII apostrophe",
+						"regex": "[’]",
+						"correction": {
+							"replaceWith": "'"
+						}
+					},
+					{
+						"description": "non-ASCII quotation mark",
+						"regex": "[“]",
+						"correction": {
+							"replaceWith": "\""
+						}
+					},
+					{
+						"description": "non-ASCII character",
+						"regex": "[^\\x00-\\x7F]+"
+					},
+					{
+						"description": "missing space before commment",
+						"regex": "(^\\s*#+)([^\\s#])",
+						"isError": false,
+						"correction": {
+							"replaceWith": "\\1 \\2"
+						}
+					},
+					{
+						"description": "extra space before punctuation",
+						"regex": "(?<!\\\\.*)\\s+([:,.?!;][^\\s]*)",
+						"except":["\\.[0-9]*$", "!="],
+						"correction": {
+							"replaceWith": "\\1"
+						}
+					},
+					{
+						"description": "missing space after punctuation",
+						"regex": "([^\\s]*[:,.?!;]+)(?!(?:[ \"'`\\)\\]:,.?!;<]|$))([^\\s\"`'.!?]*)",
+						"except": ["[.,][0-9]", "https?://", "[A-Z]\\.", "\\.{3}[\\w\\(\\[\"'`]", "[><]\\?=", "^[\\(\\[]?([\\w-]+\\.)*[\\w-]*[\\)\\]]?$", "^[\"`']*!"],
+						"correction": {
+							"replaceWith": "\\1 \\2"
+						}
+					},
+					{
+						"description": "unnecessarily split word (space ship -> spaceship, an other -> another)",
+						"regex": "(?<![^\\s])(space|an) (ship|other)(?![^\\s])",
```

Comment:
This also matches the line "When in space other ships start approaching you". (Which shouldn't be rejected on a split word check.)

Reply:
Yes, it does match. If you check the next line however, you can see that it excludes anything other than 'space ship' or 'an other'. I know it feels convoluted to check for them twice, but this is the most expandable pattern I could find (it is really easy to add other two-part entries without creating a whole new check).

At least, it is supposed to work this way. Did you get a false positive?

Reply:
No I didn't get a false positive. This is okay this way.